react/src/base/inputs/SprkCheckbox.stories.js

```diff
@@ -0,0 +1,533 @@
+import SprkErrorContainer from './SprkErrorContainer/SprkErrorContainer';
+import SprkCheckboxGroup from './SprkCheckbox/SprkCheckboxGroup/SprkCheckboxGroup';
+import SprkCheckboxItem from './SprkCheckbox/SprkCheckboxItem/SprkCheckboxItem';
+import SprkFieldset from './SprkFieldset/SprkFieldset';
+import SprkLegend from './SprkLegend/SprkLegend';
+import SprkHelperText from './SprkHelperText/SprkHelperText';
+import SprkStack from '../../objects/stack/SprkStack';
+import SprkStackItem from '../../objects/stack/components/SprkStackItem/SprkStackItem';
+import { markdownDocumentationLinkBuilder } from '../../../../storybook-utilities/markdownDocumentationLinkBuilder';
+
+export default {
+  title: 'Components/Input/Checkbox',
+  decorators: [(story) => <div className="sprk-o-Box">{story()}</div>],
+  component: SprkCheckboxGroup,
+  parameters: {
+    subcomponents: {
+      SprkCheckboxGroup,
+      SprkCheckboxItem,
+      SprkFieldset,
+      SprkLegend,
+      SprkHelperText,
+      SprkSelectionInput,
+    },
+    jest: [
+      'SprkErrorContainer',
+      'SprkCheckboxGroup',
+      'SprkCheckboxItem',
+      'SprkFieldset',
+      'SprkLegend',
+      'SprkHelperText',
+    ],
+    info: `
+    ${markdownDocumentationLinkBuilder('input')}
+- Spark checkboxes have been refactored into a compositional style component.
+The version that uses <SprkSelectionInput> will be permanently removed
+in our Fall 2020 release.
+- To update to the new version, replace checkboxes implemented by
+the <code>SprkSelectionInput</code> component with our
+new implementation using <code>SprkCheckboxGroup</code>
+and <code>SprkCheckbox</code> components.
+    `,
+  },
+};
+export const defaultStory = () => (
+  <SprkCheckboxGroup>
+    <SprkFieldset>
+      <SprkLegend>Group Label Name</SprkLegend>
+      <SprkCheckboxItem>Checkbox Item 1</SprkCheckboxItem>
+      <SprkCheckboxItem>Checkbox Item 2</SprkCheckboxItem>
+      <SprkCheckboxItem>Checkbox Item 3</SprkCheckboxItem>
+    </SprkFieldset>
+  </SprkCheckboxGroup>
+);
+
+defaultStory.story = {
+  name: 'Default',
+  parameters: {
+    jest: ['SprkCheckboxGroup'],
+  },
+};
+
+export const defaultHelperText = () => (
+  <SprkCheckboxGroup>
+    <SprkFieldset>
+      <SprkLegend>Group Label Name</SprkLegend>
+      <SprkCheckboxItem>Checkbox Item 1</SprkCheckboxItem>
+      <SprkCheckboxItem>Checkbox Item 2</SprkCheckboxItem>
+      <SprkCheckboxItem>Checkbox Item 3</SprkCheckboxItem>
+    </SprkFieldset>
+    <SprkHelperText>
+      Optional helper text, used to clarify the field&#x27;s intent
+    </SprkHelperText>
+  </SprkCheckboxGroup>
+);
+
+defaultHelperText.story = {
+  name: 'Default Helper Text',
+  parameters: {
+    jest: ['SprkCheckboxGroup'],
+  },
+};
+
+export const invalidCheckbox = () => (
+  <SprkCheckboxGroup>
+    <SprkFieldset>
+      <SprkLegend>Group Label Name</SprkLegend>
+      <SprkCheckboxItem ariaDescribedBy="checkbox-error-container">
```

Comment:
hey @RVMendoza , if the component adds this automatically, we dont need to add it to the copy code.

Reply:
The ariaDescribedBy needs to match the id of the error container, that's why I added it to the copy code.
![image](https://user-images.githubusercontent.com/4342363/85600754-bbd6a500-b61b-11ea-994f-a3f5ba683880.png)

Reply:
@RVMendoza , doesn't the Checkbox group take care of that?

Reply:
The error should be outside of the fieldset too

Reply:
done!